frontend/components/Meetups/MeetupsList.vue

```diff
@@ -42,7 +42,7 @@ const meetupTags = ( meetup ) => {
 
 </script>
 <template>
-  <LoadingSpinner v-if="loading" class="bg-zinc-100"/>
+  <MeetupsListSkeleton v-if="loading"/>
```

Comment:
Maybe `MeetupsPlaceholder` would be more verbose?

Reply:
Sure, I just changed it

Reply:
@krzysztofrewak can we resolve this conversation?:)